components/gcp-click-to-deploy/src/configs/env-kubeflow.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Why do we need this file?

Reply:
We don't; side effect of copying docs/gke/configs over.
I removed it.